### ISSUE TITLE
Spatial crop slicing into spatial augmented tensor only

### ISSUE THREAD

**Issue by thibo73800** [Visual-Behavior/aloception#13]
* the drop method must be call only if the associated label implement the crop method
* Also the crop method should handler partially None slice (start=int, end=None)

**Comment by ragier** [comment]
How to reproduce : 
```
import torch
from aloscene.tensors import SpatialAugmentedTensor

x = torch.rand((1, 3, 3, 4))
y = SpatialAugmentedTensor(x, names=("B", "C", "H", "W"))

y2 = SpatialAugmentedTensor(x, names=("B", "C", "H", "W"))

y.add_child("mychild", y2)

y = y[:, :, 1:, 1:]
```

**Comment by ragier** [comment]
@thibo73800 what does it mean : "the crop method must be call only if the associated label implement the crop method"
You want to silently stop recursion if not implemented?
